include/openssl/ssl.h.in

```diff
@@ -654,16 +654,16 @@ void SSL_set_msg_callback(SSL *ssl,
         SSL_ctrl((s),SSL_CTRL_GET_EXTMS_SUPPORT,0,NULL)
 
 # ifndef OPENSSL_NO_SRP
-
 /* see tls_srp.c */
+#  ifndef OPENSSL_NO_DEPRECATED_3_0
```

Comment:
This seems to be really out of place in this PR?

Reply:
The fix
```
commit de81f1cc3243ddcefb2cba92823206f9afb1c9db
Author: Dr. David von Oheimb <David.von.Oheimb@siemens.com>
Date:   Sat Mar 20 13:57:08 2021 +0100

    ssl.h.in: Fix deprecation exclusion for SRP-related declarations
```
adapting `include/openssl/ssl.h.in` and `util/libssl.num` came up while I prepared this PR.
All right, moved to a separate PR.